deploy/releases/v0.0.4/deployment.yaml

```diff
@@ -33,7 +33,7 @@ spec:
         - key: "node-role.kubernetes.io/master"
           effect: NoSchedule
       containers:
-      - image: packethost/packet-cloud-controller-manager:latest
+      - image: packethost/packet-ccm:v0.0.4
```

Comment:
Nice catch, thank you @ashleyschuett . Originally the image was named `packet-cloud-controller-manager`, but decided to sync the name with the GitHub repo name later. Thanks!